Sming/third-party/.patches/axtls-8266.patch

```diff
@@ -196,6 +222,24 @@ index 0cf2e4c..78576f7 100644
      if (IS_SET_SSL_FLAG(SSL_SENT_CLOSE_NOTIFY))
          return SSL_CLOSE_NOTIFY;
  
+@@ -2226,8 +2229,15 @@ void DISPLAY_STATE(SSL *ssl, int is_send, uint8_t state, int not_ok)
+     if (!IS_SET_SSL_FLAG(SSL_DISPLAY_STATES))
+         return;
+ 
+-    printf(not_ok ? "Error - invalid State:\t" : "State:\t");
```

Comment:
Not completely agree with you here. To me the effort of 2 minutes more is worth the extra 50 bytes of saved RAM because your proposed change will store "%s" in flash and the other const char strings in RAM...